include/sgx-util/bytes.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#if(FAASM_SGX_ATTESTATION)
+#ifdef __cplusplus
+extern "C" {
+#endif
+    void hex_decode(const char *in, size_t len, uint8_t *out);
+#ifdef __cplusplus
+};
+#endif
+#endif
+
+namespace util {
+    std::vector<uint8_t> stringToBytes(const std::string &str);
+
+    std::string bytesToString(const std::vector<uint8_t> &bytes);
+
+    void trimTrailingZeros(std::vector<uint8_t> &vectorIn);
+
+    int safeCopyToBuffer(const std::vector<uint8_t> &dataIn, uint8_t *buffer, int bufferLen);
+
+    int safeCopyToBuffer(const uint8_t *dataIn, int dataLen, uint8_t *buffer, int bufferLen);
```

Comment:
This is duplicating a functionality that already exists in faabric (https://github.com/faasm/faabric/blob/master/src/util/bytes.cpp)